docs/package.json

```diff
@@ -46,7 +46,10 @@
     "mocha": "^5.0.1",
     "node-sass": "^4.9.3",
     "node-static": "^0.7.11",
+    "preact": "^10.0.0-rc.1",
     "preact-cli": "^3.0.0-next.14",
+    "preact-render-to-string": "^4.1.0",
```

Comment:
This has only to be fixed with the new `preact-cli` version (in should be in `2.0`)